### PR TITLE
[RUIN-302] Adding correct google form links

### PR DESCRIPTION
**Pull request by CaseyJMay**
## Testing

1. Check out this branch.
2. Check to see that the form links all look good where they are.
3. Try and click on a few form links (welcome page, home page, questions form, basically everywhere) and make sure it takes you to the right form.
4. That's it! Let me know in the comments what was good/bad.

### CODE REVIEW

File: components/home/Home.js [hunk 1]
```diff
@@ -210,6 +210,12 @@ class Home extends Component {
                     </Section> : null
                     }
                 </ScrollView>
```
Comment (KristinAoki) [inline components/home/Home.js:212]: Add Divider to distinguish between the footer and central part of the screen.

File: components/home/Home.js [hunk 2]
```diff
@@ -210,6 +210,12 @@ class Home extends Component {
...
+                <VStack alignItems="center" mb={8}>
+                    <TouchableOpacity onPress={() => Linking.openURL('https://forms.gle/aXVjxVrQU6jm3KUx6')}><Text style={{ color: 'blue' }}>Submit Feedback</Text></TouchableOpacity>
+                    <Text textAlign="center">
+                      {"Built by students at Olin College of Engineering in partnership with the Volpe Center and Santos Family Foundation"}
```
Comment (KristinAoki) [inline components/home/Home.js:216]: Seems like a lot to have this on every single screen

File: components/modules/QuestionForm.js [hunk 3]
```diff
@@ -277,6 +279,12 @@ const QuestionForm = (props) => {
       <Box p={4}>
```
Comment (KristinAoki) [inline components/modules/QuestionForm.js:279]: might want to add a top margin to this button. Spacing looks weird with addition text underneath

File: components/modules/quickSurvey/QuickSurvey.js [hunk 4]
```diff
@@ -354,6 +355,12 @@ class QuickSurvey extends Component {
            </VStack>
        </ScrollView>
        <Button onPress={() => moveHome() } mb={4} mx={4}>Continue</Button>
```
Comment (KristinAoki) [inline components/modules/quickSurvey/QuickSurvey.js:357]: Refer to comment about Save Progress button in Question Forms

File: screens/finalReportScreen.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import FinalReport from '../components/FinalReport';
-import { View, ScrollView, Keyboard, BackHandler, Pressable } from 'react-native';
+import { View, ScrollView, Keyboard, BackHandler, Pressable, Linking } from 'react-native';
```
Comment (KristinAoki) [inline screens/finalReportScreen.js:3]: This import should happen in FinalReport.js

File: components/FinalReport.js [hunk 6]
```diff
@@ -124,6 +125,12 @@ class FinalReport extends Component {
                         this.state.exportAction(result.selectedItem.label.substring(1));
                     }}
               />
+              <VStack alignItems="center" mb={8}>
```
Comment (KristinAoki) [inline components/FinalReport.js:128]: This placement does not match all the other screens. Might consider adding a `ScrollView` tags and placing this outside the tag. Reference how the other pages are laid out.

**Review by KristinAoki — CHANGES_REQUESTED** [review]
Please review all comments. One thing to consider is creating a footer component and importing that on all of the screens. This will greatly reduce the amount of copying and pasting on every screen.

File: components/home/Home.js (hunk 2 above)
Comment (CaseyJMay) [reply components/home/Home.js:216]: Roger that, I'll remove it from most of them

Comment (KristinAoki) [reply components/home/Home.js:216]: Definitely leave it on the Welcome Screen. Maybe include it on the Final Report screen

**Comment by KristinAoki** [comment]
The footer also needs to be added to `SaveToDevice.js` and `EmailFinalReport.js`

File: components/home/Home.js (hunk 1 above)
Comment (CaseyJMay) [reply components/home/Home.js:212]: might need feedback tomorrow during the day to determine what this looks like, not super good with styling

**Comment by CaseyJMay** [comment]
@KristinAoki I responded to all of these other than the thought about creating a footer component. Maybe we can chat today about whether I should focus on that/how easy it is! If it's fairly quick, I'm down, otherwise I should probably prioritize other things :)

File: components/EmailFinalReport.js [hunk 7]
```diff
@@ -1,7 +1,8 @@
 import React, {Component} from 'react';
 import { SafeAreaView } from 'react-navigation';
 import { connect } from 'react-redux';
-import { StyleSheet, Alert, View, Dimensions } from 'react-native';
+import { StyleSheet, Alert, View, Dimensions, VStack, Linking, ScrollView } from 'react-native';
```
Comment (KristinAoki) [inline components/EmailFinalReport.js:4]: VStack is imported from `native-base` not `react-native`

File: components/SaveToDevice.js [hunk 8]
```diff
@@ -153,6 +156,10 @@ componentDidMount() {
             Your report did not save successfully. Please try again later.
           </Text>
         </MaterialDialog>
+            <SafeAreaView alignItems = "center">
```
Comment (KristinAoki) [inline components/SaveToDevice.js:159]: Do you want this placement to match EmailReport?

File: components/SaveToDevice.js [hunk 9]
```diff
@@ -153,6 +156,10 @@ componentDidMount() {
...
+            <TouchableOpacity onPress={() => Linking.openURL('https://forms.gle/aXVjxVrQU6jm3KUx6')}><Text style={{ color: 'blue' }}>Submit Feedback</Text></TouchableOpacity>
```
Comment (KristinAoki) [inline components/SaveToDevice.js:160]: Have you tried following the same technique used on the Welcome screen to force the footer to the bottom?

**Review by KristinAoki — APPROVED** [review]

**Pull request merged**